boa/src/builtins/string/mod.rs

```diff
@@ -1162,9 +1162,9 @@ impl String {
         };
         let length = primitive_val.chars().count() as i32;
         // If less than 2 args specified, end is +infinity, the maximum number value.
-        // Using i32::max_value() should be safe because the final length used is at most
+        // Using i32::MAX should be safe because the final length used is at most
         // the number of code units from start to the end of the string,
-        // which should always be smaller or equals to both +infinity and i32::max_value
+        // which should always be smaller or equals to both +infinity and i32::MAX
         let end = if args.len() < 2 {
             i32::MAX
```

Comment:
Actually, here I think it would be best to use `i32::max_value()` in the implementation (as it was in the comment), since it's a const function.

Reply:
I didn't change anything here, it's just my fork getting old I guess